panel/tests/chat/test_interface.py

```diff
@@ -382,6 +382,28 @@ def post_callback(instance, event):
         assert chat_interface.objects[1].object == "2"
         assert chat_interface.objects[2].object == "3"
 
+    def test_custom_js_no_code(self):
+        chat_interface = ChatInterface()
+        with pytest.raises(ValueError, match="A 'code' key is required for"):
+            chat_interface.button_properties={
+                "help": {
+                    "icon": "help",
+                    "js_on_click": {
+                        "args": {"chat_input": chat_interface.active_widget},
+                    },
+                },
+            }
+
+    def test_custom_js_not_dict(self):
```

Comment:
Wouldn't be opposed to allowing this as well since providing `args` is often not necessary. Up to you.